helm/templates/receive/service-udp.yaml

```diff
@@ -1,17 +1,17 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "hindsight.fullname" . }}-receive-tcp
+  name: {{ include "hindsight.fullname" . }}-receive-udp
```

Comment:
Why the switch from tcp to udp here?

Reply:
see my comment on the `receive/deployment.yaml` file above.